### ISSUE TITLE
Usage Accounting (costs $) charts are not displaying

### ISSUE THREAD

**Issue by ppodbielski** [rchakode/kube-opex-analytics#87]
**Describe the bug**
Usage Accounting (costs $) charts are not displaying:

```
2022-11-29 11:32:59,530 - kube-opex-analytics - ERROR - KeyError Exception in dump_analytics => Traceback (most recent call last):
  File "./backend.py", line 908, in dump_analytics
    Rrd.dump_histogram_analytics(dbfiles=ns_dbfiles, period=RrdPeriod.PERIOD_14_DAYS_SEC)
  File "./backend.py", line 763, in dump_histogram_analytics
    usage_ratio * usage_per_type_date[res][date_key][KOA_CONFIG.db_billing_hourly_rate],
KeyError: '.billing-hourly-rate'
```
404 In browser:
```
https://kube-opex-analytics-url/static/data/cpu_usage_period_1209600.json?_=1669722368117
https://kube-opex-analytics-url/static/data/memory_usage_period_1209600.json?_=1669722368118
```

Configuration:
k8s: 1.22.15
k8s.gcr.io/metrics-server/metrics-server:v0.6.1

```
KOA_COST_MODEL=CHARGE_BACK
KOA_BILLING_HOURLY_RATE=6.00
```

Steps to reproduce:
1. set KOA_COST_MODEL to CHARGE_BACK and KOA_BILLING_HOURLY_RATE to 6.00
2. Deploy kube-analytics-tool in k8s

**Comment by rchakode** [comment]
Hi @ppodbielski ,
Thank to report this issue. 
We'll try to reproduce with the provided inputs.

R.

**Comment by rchakode** [comment]
Hi @ppodbielski 
This issue is fixed in the version 2022.12.1 just released.
